client/web/src/enterprise/batches/Branch.tsx

```diff
@@ -54,7 +56,10 @@ export const BranchMerge: React.FunctionComponent<React.PropsWithChildren<Branch
 }) => (
     <div className="d-block d-sm-inline-block">
         <Branch name={baseRef} />
-        <span className="p-1">&larr;</span>
+        <VisuallyHidden>Update with</VisuallyHidden>
+        <span className="p-1" aria-hidden={true}>
+            &larr;
+        </span>
```

Comment:
```suggestion
        <Icon as="span" inline={false} className="p-1" aria-label="Update with">
            &larr;
        </Icon>
```

Let's use `Icon` from Wildcard here, and add an `aria-label` instead of using VisuallyHidden. You might need to update this branch with `main` as the `inline` prop has recently been added